bin/pycbc_optimal_snr

```diff
@@ -170,13 +176,29 @@ if __name__ == '__main__':
     def get_injection(injections, det, injection_time, simulation_id):
         """ Do an injection from the injection XML file, specified by
         IFO and end time"""
-        # leave 4 s of padding at the end for possible ringdown
-        start_time = int(injection_time + 4. - seg_len)
-        strain = TimeSeries(zeros(tlen, dtype=float32), delta_t=delta_t,
+        global tlen, delta_t, sample_rate
+        if optimize_sample_rate:
+            estimate = (pycbc.pnutils.get_final_freq('SEOBNRv2', 1, 100, 1, 1) + 16 ) * 2.1
```

Comment:
Hmm are the fixed mass and spins intended? I thought the benefit would be in using the injection parameters.